readme.md

```diff
@@ -166,6 +166,7 @@
 - [ActionScript 3](https://github.com/robinrodricks/awesome-actionscript3) - Object-oriented language targeting Adobe AIR.
 - [Eta](https://github.com/sfischer13/awesome-eta) - Functional programming language for the JVM.
 - [Idris](https://github.com/joaomilho/awesome-idris) - General purpose pure functional programming language with dependent types influenced by Haskell and ML.
+- [WebAssembly](https://github.com/mbasso/awesome-wasm)
```

Comment:
> The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**